README.md

```diff
@@ -40,75 +39,42 @@ The installation procedure has been tested on Ubuntu 20.04 and Windows 10.
 On **Linux** or **macOS**, run:
 
 ```bash
+mamba activate  <conda-environment-name>
 cd ironcub_mk1_software
 mkdir build
 cd build
-cmake .. -DCMAKE_INSTALL_PREFIX="/path/to/desired/install/dir" .
+cmake .. -DCMAKE_INSTALL_PREFIX=</path/to/desired/install/dir>.
 make install
+echo "source </path/to/desired/install/dir>/share/ironcub/setup_v1.sh" > "${CONDA_PREFIX}/etc/conda/activate.d/ironcub_activate.sh"
+chmod +x "${CONDA_PREFIX}/etc/conda/activate.d/ironcub_activate.sh"
```

Comment:
Nitpick: I know that I am the one that suggested that scheme `<project>_activate.sh` . However, that is the exact same scheme used by conda packages, so if it actually works probably it would make sense to use `ironcub_setup.sh` to avoid conflicts with an `ironcub` conda package n the future.

Reply:
Fixed in 0a48574de8760f848ecc3babcdc6e2385e2dba1c